formatter/README.md

```diff
@@ -0,0 +1,148 @@
+# AUTOMATED CODE FORMATTING
+
+## BACKGROUND
+
+Code Formatting is considered "best practice" as it provides code quality assurance for a given project. Project code is consistently formatted in the same manner throughout the project tree and all development is directed to the same formatting standard. Linting and formatting source code files prior to committing to a CVS also removes extraneous whitespace characters. Differences in formatting style and extraneous whitespaces typically creep into a project over time where multiple developers are working on a project and each developer is using their preferred editing application. Problem is compounded when developers are using multiple platforms (i.e. Unix, Linux, Windows, et al)
+
+All necessary configuration settings for automated code formatting as it applies to **DB Browser for SQLite** are stored in this folder.
+
+## DEPENDENT APPLICATIONS
+
+Application leverages Python 3 to execute other applications to perform automated code formatting. These dependent applications are available across may platforms Ensure these applications are installed prior to attempting to follow the **OPERATION** instructions below.
```

Comment:
may platforms -> ma**n**y platforms**.**

Reply:
Yup. A typo. I should be pleased that someone read the docs. :)